admin/src/lib/cookies.ts

```diff
@@ -0,0 +1,14 @@
+// @ts-ignore
```

Comment:
Why do we use `@ts-ignore` here?

Reply:
If @ts-ignore is not here, an error occurs and no import is imported.